ngx_http_pipelog_module.c

```diff
@@ -2188,17 +2203,24 @@ ngx_http_pipelog_logger_process_main (ngx_cycle_t *cycle) {
     struct timespec timeout;
     int sig;
 
+    ngx_log_error(NGX_LOG_DEBUG, cycle->log, 0, "%s: start logger process %d", MODULE_NAME, getpid());
+
     memset(&sa, 0, sizeof(sa));
+
     sa.sa_handler = SIG_IGN;
     sigaction(SIGPIPE, &sa, NULL);
     sigemptyset(&set);
     sigaddset(&set, SIGCHLD);
     sigprocmask(SIG_BLOCK, &set, NULL);
+
+    sigaddset(&set, SIGTERM);
+    sigaction(SIGTERM, &sa, NULL);
```

Comment:
It seems to me that SIGTERM should just be blocked by sigprocmask() along with SIGCHLD.

Reply:
I agree. If you want to fix it or wait until I have some additional time to look over it, that's fine with me.